addons/im_livechat/tests/test_get_discuss_channel.py

```diff
@@ -52,6 +52,7 @@ def test_channel_get_livechat_visitor_info(self):
         self.assertEqual(channel_info['channel']['channelMembers'], [('insert', [
             {
                 'channel': {'id': channel_info['id']},
+                'create_date': channel_info['channel']['channelMembers'][0][1][0]['create_date'],
```

Comment:
This is cheating (comparing it against itself), it should at least get the date from the member and compare that, there is already a search to find the member below.